src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -265,6 +268,57 @@ class PyMobile:
+        pass
+
+    def SetBorderSize(self, borderSize: int) -> None:
+        """
+         Sets the border size of the nine-slice
+        
+        """
+        pass
+
+class ModernNineSliceGump:
+
+    def SetResizeCallback(self, callback: Any) -> None:
```

Ruff (0.13.1)

Undefined name `Any`

(F821)